tests/Makefile

```diff
@@ -3,11 +3,11 @@ all: test
 test: test-build
 
 test-build:
-	@btest -j -d
+	@btest -d
```

Comment:
I suppose this is necessary even thought we already have `HILTI_JIT_SEQUENTIAL` set?

Reply:
Yes, unfortunately it looks like it. When I first tried only setting the memory, the _standalone_ci_ task was still OOM'ing. I am not sure why this seems to be less of an issue for zeek/spicy's CI (maybe it has to do with us using more cpus there).